coderd/coderd.go

```diff
@@ -679,6 +679,10 @@ func New(options *Options) *API {
 		DERPFn:   api.DERPMap,
 		Logger:   options.Logger,
 		ClientID: uuid.New(),
+		DatabaseHealthcheckFn: func(ctx context.Context) error {
```

Comment:
Self-review: I didn't want to pull the whole store into the dialer, so opted for this approach.

Reply:
Good idea 👍 will address.